src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,11 +2,15 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationService.class);
```

Comment:
```suggestion
    private static final Logger logger = LogManager.getLogger(AuthenticationService.class);
```
Есть разница что здесь передавать AuthenticationService или AuthenticationServiceImpl?